File: Projects/UniverseExplorer/app/src/main/java/a45125/myapplication/MusicService.java
package a45125.myapplication;

import android.app.Service;
import android.content.Intent;
import android.media.MediaPlayer;
import android.os.IBinder;
import android.widget.Toast;

import androidx.annotation.Nullable;

public class MusicService extends Service {

    MediaPlayer musicPlayer;

    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }

    @Override public void onCreate() {
        Toast.makeText(this, "Music Service Created", Toast.LENGTH_SHORT).show();
        musicPlayer = MediaPlayer.create(this, R.raw.music);
        musicPlayer.setLooping(true);
    }

    @Override public int onStartCommand(Intent intent, int flags, int startid) {
        Toast.makeText(this, "Music Service Started", Toast.LENGTH_SHORT).show();
        musicPlayer.start(); return super.onStartCommand(intent, flags, startid);
    }

    @Override public void onDestroy() {
        Toast.makeText(this, "Music Service Stopped", Toast.LENGTH_SHORT).show();
        musicPlayer.stop();
    }
}

File: Projects/UniverseExplorer/app/src/main/java/a45125/dam/universeexplorer/MusicService.kt
package a45125.dam.universeexplorer

import android.app.Service
import android.content.Intent
import android.media.MediaPlayer
import android.os.IBinder
import android.widget.Toast

class MusicService : Service() {
    lateinit var musicPlayer: MediaPlayer
    override fun onBind(intent: Intent): IBinder? {
        return null
    }

    override fun onCreate() {
        Toast.makeText(this, "Music Service Created", Toast.LENGTH_SHORT).show()
        musicPlayer = MediaPlayer.create(this, R.raw.music)
        musicPlayer.setLooping(true)
    }

    override fun onStartCommand(intent: Intent, flags: Int, startid: Int): Int {
        Toast.makeText(this, "Music Service Started", Toast.LENGTH_SHORT).show()
        musicPlayer!!.start()
        return super.onStartCommand(intent, flags, startid)
    }

    override fun onDestroy() {
        Toast.makeText(this, "Music Service Stopped", Toast.LENGTH_SHORT).show()
        musicPlayer!!.stop()
    }
}